.gitignore

```diff
@@ -156,3 +156,5 @@ $RECYCLE.BIN/
 
 # Mac desktop service store files
 .DS_Store
+
+/PackagingProjects/CSharpPluginPayload/TempJarDir
```

Comment:
Miscellaneous: git-ignore the temporary directory created by the packaging project